packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [34.0.0]
+
+### Changed
+
+- Enable `resetMocks` Jest configuration option ([#4417](https://github.com/MetaMask/core/pull/4417))
+- **BREAKING** Update Assets Controllers ([#4219](https://github.com/MetaMask/core/pull/4219))
+  - `TokenBalancesController` messenger must allow the action `AccountsController:getSelectedAccount` and remove `PreferencesController:getState`.
+  - `TokenDetectionController` messenger must allow the action `AccountsController:getAccount`.
+  - `TokenDetectionController` messenger must allow the event `AccountsController:selectedEvmAccountChange` and remove `AccountsController:selectedAccountChange`.
+  - `TokenDetectionController` removes `selectedAddress` constructor argument.
+  - `TokenRatesController` messenger must allow the action `AccountsController:getAccount`, `AccountsController:getSelectedAccount` and remove `PreferencesController:getState`.
+  - `TokenRatesController` messenger must allow the event `AccountsController:selectedEvmAccountChange` and remove `PreferencesController:stateChange`.
+  - `TokensController` messenger must allow the action `AccountsController:getAccount`, `AccountsController:getSelectedAccount`.
+  - `TokensController` messenger must allow the event `AccountsController:selectedEvmAccountChange`.
+  - `TokensController` removes `selectedAddress` constructor argument.
+- Upgrade AccountTrackerController to BaseControllerV2 ([#4407](https://github.com/MetaMask/core/pull/4407))
```

Comment:
There are a lot of changes in this PR we can capture here.

I think we'd want to add these to Added:

``` markdown
- Add `AccountTrackerControllerGetStateAction`, `AccountTrackerControllerActions`, `AccountTrackerControllerStateChangeEvent`, and `AccountTrackerControllerEvents` types ([#4407](https://github.com/MetaMask/core/pull/4407))
- Add `setIntervalLength` and `getIntervalLength` methods to `AccountTrackerController` ([#4407](https://github.com/MetaMask/core/pull/4407))
  - `setIntervalLength` replaces updating the polling interval via `configure`.
```

And add these to Changed:

``` markdown
- **BREAKING:** Convert `AccountInformation` from interface to type ([#4407](https://github.com/MetaMask/core/pull/4407))
- **BREAKING:** Rename `AccountTrackerState` to `AccountTrackerControllerState` and convert from interface to type ([#4407](https://github.com/MetaMask/core/pull/4407))
- **BREAKING:** `AccountTrackerController` now inherits from `StaticIntervalPollingController` instead of `StaticIntervalPollingControllerV1` ([#4407](https://github.com/MetaMask/core/pull/4407))
  - The constructor now takes a single options object rather than three arguments. Some options have been removed; see later entries.
- **BREAKING:** The `AccountTrackerController` messenger must now allow the actions `PreferencesController:getState`, `NetworkController:getState`, and `NetworkController:getNetworkClientById` ([#4407](https://github.com/MetaMask/core/pull/4407))
- **BREAKING:** The `refresh` method is no longer pre-bound to the controller ([#4407](https://github.com/MetaMask/core/pull/4407))
  - You may now need to pre-bind it e.g. `accountTrackerController.refresh.bind(accountTrackerController)`.
```

Add add these to Removed:

``` markdown
- **BREAKING:** Remove `AccountTrackerConfig` type ([#4407](https://github.com/MetaMask/core/pull/4407))
  - Some of these properties have been merged into the options that the `AccountTrackerController` constructor takes.
- **BREAKING:** Remove `config` property and `configure` method from `AccountTrackerController` ([#4407](https://github.com/MetaMask/core/pull/4407))
  - The controller now takes a single options object which can be used for configuration, and configuration is now kept internally.
- **BREAKING:** Remove `notify`, `subscribe`, and `unsubscribe` methods from `AccountTrackerController` ([#4407](https://github.com/MetaMask/core/pull/4407))
  - Use the controller messenger for subscribing to and publishing events instead.
- **BREAKING:** Remove `provider`, `getMultiAccountBalancesEnabled`, `getCurrentChainId`, and `getNetworkClientById` from configuration options for `AccountTrackerController` ([#4407](https://github.com/MetaMask/core/pull/4407))
  - The provider is now obtained directly from the network controller on demand.
  - The messenger is now used in place of the callbacks.
```

Reply:
Sure, let me add these.